### DISCUSSION TITLE
Voice clone

### DISCUSSION THREAD

**Discussion by LawStudent99** [suno-ai/bark#529] [General]
Hi 

Is it possible to make voice replicas with Suno/Bark? 

I am aware that the programme itself doesn't allow voice cloning. Is it however possible to somehow bypass their restrictions? 

I would be pleased if someone could help me on this as I need the information for a research project. 

Thank you.

**Comment by prizrak2033** [comment]
I'm looking into this myself; I'm trying to use it for an OSINT hackathon challenge this semester.

**Reply by LawStudent99** [reply]
If you succeed, I would be pleased if you would let me know.
